sopel/modules/wikipedia.py

```diff
@@ -88,7 +88,10 @@ class WikipediaSection(StaticSection):
     default_lang = ValidatedAttribute('default_lang', default='en')
     """The default language to find articles from (same as Wikipedia language subdomain)."""
     lang_per_channel = ValidatedAttribute('lang_per_channel')
-    """List of ``#channel:langcode`` pairs to define Wikipedia language per channel."""
+    """List of ``#channel:langcode`` pairs to define Wikipedia language per channel.
```

Comment:
How was that supposed to work with a ValidatedAttribute before that? Like a list attribute (separated by new-lines)?

I don't think it needs to be modified, but it's... weird.

Reply:
Definitely not going to change how it works at the same time as deprecating it. 😅

Anyway, you can see in the deleted example under `configure()` that it was a plain old `ValidatedAttribute` that kinda-sorta looked like a `ListAttribute` but "parsed" by the regex search I moved into `choose_lang()` instead of by splitting on `,`.

Yes it was a weird way of doing things. Yes that's one of the reasons I'm quite happy to deprecate that mess. 😉